plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -508,7 +509,8 @@ public Stream<TableColumnsMetadata> streamTableColumns(ConnectorSession session,
                 LOG.debug(e, "Ignored exception when trying to list columns from %s", table);
                 return Stream.empty();
             }
-        });
+        })
+                .iterator();
```

Comment:
Put on previous line

Reply:
the stream operations like map or iterator typically go on individual lines, i would rather adhere to that